gcp/live/dev/k8s/gpii/couchdb/terraform.tfvars

```diff
@@ -18,3 +18,7 @@ terragrunt = {
 }
 
 # ↓ Module configuration (empty means all default)
+
+couchdb_replicas = 2
+backup_deltas = "PT5M PT15M PT45M"
```

Comment:
Where are the stg (and prd) values?

Reply:
We don't have prd in `live` yet, and what we have for stg is seriously outdated. Do you think I should work on this here?

Reply:
Yes.

Reply:
Ok, updated stg/prd couchdb Terragrunt modules.